src/NuGet.Clients/NuGet.VisualStudio.Contracts/GetInstalledPackageResultStatus.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.VisualStudio.Contracts
+{
+    /// <summary>The status of the result</summary>
+    public enum GetInstalledPackageResultStatus
```

Comment:
I thought we were leaning toward InstalledPackageResultStatus, removing the get?